src/gsi/windht.f90

```diff
@@ -101,6 +101,12 @@ subroutine readin_windht_list(filename,fexist,ncount)
        ncount=n-1
        print*, "Number of provider/subprovider combinations:",ncount
        close(meso_unit)
+
+!      print out the list
+!      do n = 1, ncount
+!         write(6,'(1x,A,2X,A16,2X,F5.2,2x,I3.3)')'readin_windht_list::prov+subrpov height:',provlist(n), heightlist(n), n
+!      end do
+
```

Comment:
Please clean those commented code.

Reply:
@hu5970 
Hi Ming,

I am going to clean these commented lines very soon.

Thank you!

-Gang

Reply:
@hu5970 
Hi Ming,

Those commented lines were cleaned off. You may go ahead check the updated code now.
Thank you!

-Gang